core/src/main/java/io/aiven/klaw/config/MigrationUtility.java

```diff
@@ -44,18 +45,18 @@ public void afterPropertiesSet() throws Exception {
 
     // Find the latest version in DB
     String latestDataVersion = getLatestDataVersion();
-    if (!isVersionGreaterThenOrEqualToCurrentVersion(latestDataVersion, currentKlawVersion)) {
+    if (!isVersionGreaterThenCurrentVersion(latestDataVersion, currentKlawVersion)) {
```

Comment:
In general, after migration, can we drop the column otherparams from kwenv ? Or reuse the same column otherparams instead of new column 'envparams'

Reply:
I don't think we can do this safely in this release as if there is an issue say with a particular user, upgrading and they need to downgrade we would have deleted all their data, or if their is an issue in the migration which would get logged out they would potentially lose some data before they get a chance to rectify the issue, what we could do is say it is deprecated and due for removal in the future .

I think we have spoken about depreciation before and I think a 3-6 month lead time to dropping it would be a good idea. Then we just need to say in the release docs we are also deleting this from the table as it is deprecated.